tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Core.cs

```diff
@@ -114,12 +128,6 @@ private Dictionary<string, object> GetBasicRequestArgsForWaf()
             { AddressesConstants.RequestClientIp, _localRootSpan.GetTag(Tags.HttpClientIp) }
         };
 
-        var userId = _localRootSpan.Context?.TraceContext?.Tags.GetTag(Tags.User.Id);
```

Comment:
Dont do this anymore: this could override the sdk, we now need a more precise approach with values from the sdk not overriden and others yes